301-subnet-driven-deployment/WinServ.json

```diff
@@ -0,0 +1,403 @@
+﻿{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+  "contentVersion": "0.0.0.1",
+  "parameters": {
+    "dDisks": {
+      "type": "array",
+      "defaultValue": [ ],
+      "metadata": {
+        "description": "Array of disks objects that can be added"
```

Comment:
Added a reference to the azuredeploy instead. A sample on a single line is not that clear.